VideoLocker/res/layout/fragment_add_comment.xml

```diff
@@ -19,13 +19,14 @@
             android:paddingBottom="@dimen/widget_margin"
             android:paddingLeft="@dimen/edx_margin"
             android:paddingRight="@dimen/edx_margin"
-            android:paddingTop="@dimen/widget_margin">
+            android:paddingTop="@dimen/widget_double_margin">
```

Comment:
Why not just add a top margin to the new included layout as well, instead of having to define this new dimension?